packages/xchain-cosmos/src/client.ts

```diff
@@ -29,14 +29,20 @@ export interface CosmosClient {
   getSDKClient(): CosmosSDKClient
 }
 
-const MAINNET_SDK = new CosmosSDKClient({
-  server: 'https://api.cosmos.network',
-  chainId: 'cosmoshub-4',
-})
-const TESTNET_SDK = new CosmosSDKClient({
-  server: 'https://rest.sentry-02.theta-testnet.polypore.xyz',
-  chainId: 'theta-testnet-001',
-})
+export interface CosmosClientParams extends XChainClientParams {
+  serverUrl?: string
+}
+
+const MAINNET_SDK = (serverUrl: string | undefined) =>
+  new CosmosSDKClient({
+    server: serverUrl || 'https://api.cosmos.network',
+    chainId: 'cosmoshub-4',
+  })
+const TESTNET_SDK = () =>
```

Comment:
[nitpick] Rename `MAINNET_SDK` -> `createMainnetClient` / `TESTNET_SDK` -> `createTestnetClient` or just move initializations of `CosmosSDKClient` into `constructor` of `Client` (it's not needed anywhere else)

Reply:
Fixed in #599